paper.tex

```diff
@@ -64,6 +64,20 @@ \section{Related Work}
 
 
 \section{System Architecture}
+\subsection{Participants}
+There are four major roles in the decentralized data marketplace.
+
+\subsubsection{Registrar}
+Registrar is responsible for creating the Registration Contract and adding new data providers, consumers and brokers to the decentralized data marketplace.
```

Comment:
So, a registrar is an official keeper of records made in a register. Did I understand correctly?

Reply:
Did you mention the purpose of Registration Contract before?

Reply:
> So, a registrar is an official keeper of records made in a register. Did I understand correctly?

Yeah.

> Did you mention the purpose of Registration Contract before?

Not yet, I'll add a brief description.